src/transformStateWithClones.js

```diff
@@ -7,7 +7,36 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  let currentState = { ...state };
+
+  for (const act of actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```
What act is it, first or second? 😄